core/jvm/src/main/scala/cats/effect/unsafe/SelectorSystem.scala

```diff
@@ -161,6 +161,7 @@ final class SelectorSystem private (provider: SelectorProvider) extends PollingS
   final class Poller private[SelectorSystem] (
       private[SelectorSystem] val selector: AbstractSelector
   ) extends PollerMetrics {
+    override val classifier: String = "selector"
```

Comment:
It can just be a `def` because constant `String`s are already pre-allocated in the pool.

```suggestion
    override def classifier: String = "selector"
```